packages/qwik-city/runtime/src/link-component.tsx

```diff
@@ -19,7 +19,12 @@ export const Link = component$<LinkProps>((props) => {
           prefetchLinkResources(elm as HTMLAnchorElement, ev.type === 'qvisible')
         )
       : undefined;
-  const handleClick = event$(async (_: any, elm: HTMLAnchorElement) => {
+  const handleClick = event$(async (ev: any, elm: HTMLAnchorElement) => {
+    if (ev.ctrlKey || ev.metaKey) {
+      // Do not enter the nav pipeline if any option key is pressed.
+      return;
```

Comment:
I don't think this will work because events are handled async in Qwik.
I am working on sync events and once that lands we can include this.